### ISSUE TITLE
Add back the 'respawn' module filters

### ISSUE THREAD

**Issue by qlados** [PGMDev/PGM#246]
This filter is essential for many competitive maps. You were able to apply (any, but mostly time) filters to this module, which allowed for scaling respawn timers. 

Example:
   
```
<respawns>
    <respawn delay="3s" auto="true" blackout="true" filter="spawn-1"/>
    <respawn delay="3.5s" auto="true" blackout="true" filter="spawn-2"/>
    <respawn delay="4s" auto="true" blackout="true" filter="spawn-3"/>
    <respawn delay="4.5s" auto="true" blackout="true" filter="spawn-4"/>
    <respawn delay="5s" auto="true" blackout="true" filter="spawn-5"/>
    <respawn delay="5.5s" auto="true" blackout="true" filter="spawn-6"/>
</respawns>
<filters>
    <not id="spawn-1">
        <time>2m</time>
    </not>
    <all id="spawn-2">
        <time>2m</time>
        <not><time>4m</time></not>
    </all>
    <all id="spawn-3">
        <time>4m</time>
        <not><time>6m</time></not>
    </all>
    <all id="spawn-4">
        <time>6m</time>
        <not><time>8m</time></not>
    </all>
    <all id="spawn-5">
        <time>8m</time>
        <not><time>10m</time></not>
    </all>
    <all id="spawn-6">
        <time>10m</time>
    </all>
</filters>
```

**Comment by Pablete1234** [comment]
see: #713 & #718

**Comment by Pablete1234** [comment]
fixed in #762
